src/triggers/CBL_Attitud_UpdateDataCbl2.trigger

```diff
@@ -0,0 +1,51 @@
+trigger CBL_Attitud_UpdateDataCbl2 on CBL_Attitude__c (after insert) {
+	List<CBL_Attitude__c> cbls_att = new List<CBL_Attitude__c>();
+    Set<Id> cbls_Ids = new Set<Id>();
+    Set<Id> LAC_FD_Ids = new Set<Id>();
+    
+    //get a list with Baseline id related to attitude rows inserted
+    for (CBL_Attitude__c newRow : Trigger.new) {
+        cbls_Ids.add(newRow.Baseline__c);
+    }
+    //get a list with attitude rows completed (Baseline__r.Farmer__c)
+    cbls_att = [SELECT Id, Baseline__c, Baseline__r.Farmer__c, chooseToBeFarmer__c, destinedToBeFarmer__c, discussMethodsWOthers__c, effortInAgriculturePays__c, farmersLifeWillBeBetter__c, futureCoffeePlans__c, happyToBeFarmer__c, limitationsCauses__c, neighborsProductivityLvl__c, payForAgriAdvise__c, productivityImprovmntLimitations__c, productivityTrend5Years__c, takeAdvise__c, wishChildrenBecomeFarmers__c, worthInvestingMoneyTimeAgr__c, Submission__c 
```

Comment:
Let's chat about this - salesforce automatically batches the SOQL if you
use it in a for-loop like I mentioned.

Luke.
ᐧ


Luke Kyohere

Group Manager, Software Development

Grameen Foundation <http://www.grameenfoundation.org/>

Mobile: 646-321-2691

lkyohere@grameenfoundation.org | Skype: luke1kyohere

Facebook <https://www.facebook.com/StopPovertyNow> | Twitter
<https://twitter.com/grameenfdn> | YouTube
<https://www.youtube.com/user/grameen?blend=6&ob=5>


<https://www.youtube.com/user/grameen?blend=6&ob=5>

*Connecting the World's Poor to Their Potential*


On Sat, Feb 28, 2015 at 12:11 AM, Carlos Ardila <notifications@github.com>
wrote:

> In src/triggers/CBL_Attitud_UpdateDataCbl2.trigger
> <https://github.com/grameenfoundation/applab-ug.salesforce/pull/47#discussion_r25546807>
> :
>
> > @@ -0,0 +1,51 @@
> > +trigger CBL_Attitud_UpdateDataCbl2 on CBL_Attitude__c (after insert) {
> > +	List<CBL_Attitude__c> cbls_att = new List<CBL_Attitude__c>();
> > +    Set<Id> cbls_Ids = new Set<Id>();
> > +    Set<Id> LAC_FD_Ids = new Set<Id>();
> > +
> > +    //get a list with Baseline id related to attitude rows inserted
> > +    for (CBL_Attitude__c newRow : Trigger.new) {
> > +        cbls_Ids.add(newRow.Baseline__c);
> > +    }
> > +    //get a list with attitude rows completed (Baseline__r.Farmer__c)
> > +    cbls_att = [SELECT Id, Baseline__c, Baseline__r.Farmer__c, chooseToBeFarmer__c, destinedToBeFarmer__c, discussMethodsWOthers__c, effortInAgriculturePays__c, farmersLifeWillBeBetter__c, futureCoffeePlans__c, happyToBeFarmer__c, limitationsCauses__c, neighborsProductivityLvl__c, payForAgriAdvise__c, productivityImprovmntLimitations__c, productivityTrend5Years__c, takeAdvise__c, wishChildrenBecomeFarmers__c, worthInvestingMoneyTimeAgr__c, Submission__c
>
> if the idea is process in batches of 200, do I need to put a limit clause
> in the query?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grameenfoundation/applab-ug.salesforce/pull/47/files#r25546807>
> .
>